### PR TITLE
IGNITE-12993: Initial ASF infra integration configuration.

### PR DESCRIPTION
**Pull request by xtern**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - iot
+    - osgi
+    - network-client
+    - ignite
+    - data-management-platform
+    - big-data
+    - cloud
+    - database
+    - network-server
+    - hadoop
```
Comment (Mmuzaf) [inline .asf.yaml:30]: Do we remove Hadoop support from Ignite?

Comment (xtern) [reply .asf.yaml:30]: These are the current labels of Apache Ignite repository on github

Comment (xtern) [reply .asf.yaml:30]: I'm not sure we can remove this label without discussion.

IGFS and Ignite Hadoop Accelerator were discontinued, but Ignite can still be used to speed up Hadoop, as described on this page: https://ignite.apache.org/use-cases/hadoop-acceleration.html

**Comment by Mmuzaf** [comment]
LGTM,

@dmagda @pavlukhin 

Folks, can you take a look too?

File: .asf.yaml (hunk 1 above)
Comment (dmagda) [reply .asf.yaml:30]: Maxim, I would also add the following labels:

- cache
- in-memory computing
- in-memory database

**Pull request merged**